src/Generator.Rewrite/OpenTKAssemblyResolver.cs

```diff
@@ -53,7 +53,14 @@ public AssemblyDefinition Resolve(AssemblyNameReference name, ReaderParameters p
 
             var framework_dir = Path.GetDirectoryName(typeof(object).Module.FullyQualifiedName);
             var framework_dirs = on_mono
-                ? new[] { framework_dir, Path.Combine(framework_dir, "Facades") }
+                ? new[]
```

Comment:
Looks good. I worry that we'll have to maintain our own version of the resolver in the future because of this, though.